compose/services/docker-compose.postgis.yml

```
@@ -0,0 +1,27 @@
+services:
+  postgis:
+    container_name: postgis
+    hostname: postgis
+  postgis:
```

YAMLlint (1.35.1)

[error] duplication of key "postgis" in mapping

(key-duplicates)